### DISCUSSION TITLE
Opening the Gates of Management and Science section

### DISCUSSION THREAD

**Discussion by DrMattG** [DrMattG/TechBk1#17] [Ideas]
@ChloeRN & @ErlendNilsen 
Thinking about these two proposed sub-sections in the Opening the Gates of Management and Science section: 
- **Open source software**: do you all agree that this includes R/Python packages; But I guess it could also include stand-alone code (e.g. associated with a journal article, etc)? We can find the most downloaded wildlife/ecology focused R packages for example - but probably tough to find stand-alone code (but perhaps we can just highlight this might be a good source of code). This could also include Apps as well.
 
- **Open hardware**:  I'm thinking of things like [AudioMoth](https://doi.org/10.1016/j.ohx.2019.e00073) or [FishCam](https://doi.org/10.1016/j.ohx.2020.e00110) - the HardwareX Journal has quite a few different applications focused on biodiversity etc. We could do a very _quick & dirty_ review of published hardware that is related to wildlife/Ecology (Camera traps, sensors, traps, etc)

**Comment by ChloeRN** [comment]
@DrMattG  I think open source software definitely goes beyond R/Python packages. A lot of code is published now together with papers (either on GitHub - YAY! - or as supplementary material - NAY!) without ever becoming part of a package, and I think that constitutes an important resource. 
Besides that, there are also standalone software pieces and algorithms that are being used with and without linkage to R/Python (C++, machine learning algorithms, software dedicated to specific types of analyses such as E-SURGE and various applications for individual based models, animal models etc.). Those also matter. 

And then there are (online) recources for learning how to use software. Workshops and courses dedicated to teaching students, researchers, and practicioners about a software's capabilities and options. All the open code in the world is worth little if no one else can learn how to use it. So I'd really like to highlight some of the great initiatives out there providing open (= nearly or completely free) courses (thinking people like Olivier Gimenez, Richard McElreath, and more).
